### PR TITLE
Add prewrap prop to Box (white-space: pre-wrap)

### PR DESCRIPTION
**Pull request by Novacat**
This is my first TGUI PR, please be gentle.

Port of https://github.com/VOREStation/VOREStation/pull/10260

### CODE REVIEW

**Comment by stylemistake** [comment]
Here's a suggestion: rename it to `preserveWhiteSpace` and create a similar PR on tgstation (which is the upstream of tgui).

File: tgui/packages/tgui/interfaces/SecurityRecords.js [hunk 1]
```diff
@@ -179,8 +179,8 @@ const SecurityRecordsViewGeneral = (_properties, context) => {
         <LabeledList>
           {general.fields.map((field, i) => (
             <LabeledList.Item key={i} label={field.field}>
-              <Box height="20px" display="inline-block">
-                {field.value.split("\n").map(m => <Box key={m}>{m}</Box>)}
+              <Box height="20px" display="inline-block" prewrap>
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/SecurityRecords.js:182]: you can just use the `inline` property instead.

File: tgui/packages/tgui/components/Box.js [hunk 2]
```diff
@@ -110,6 +110,7 @@ const styleMapperByPropName = {
   bold: mapBooleanPropTo('font-weight', 'bold'),
   italic: mapBooleanPropTo('font-style', 'italic'),
   nowrap: mapBooleanPropTo('white-space', 'nowrap'),
+  prewrap: mapBooleanPropTo('white-space', 'pre-wrap'),
```
Comment (ShadowLarkens) [inline tgui/packages/tgui/components/Box.js:113]:
```suggestion
  preserveWhiteSpace: mapBooleanPropTo('white-space', 'pre-wrap'),
```

File: tgui/packages/tgui/interfaces/GeneralRecords.js [hunk 3]
```diff
@@ -202,8 +202,8 @@ const GeneralRecordsViewGeneral = (_properties, context) => {
             </LabeledList.Item>
           ))}
         </LabeledList>
-        <Section title="Employment/skills summary" level={2}>
-          {general.skills.split("\n").map(m => <Box key={m}>{m}</Box>) || "No data found."}
+        <Section title="Employment/skills summary" level={2} prewrap>
```
Comment (ShadowLarkens) [inline tgui/packages/tgui/interfaces/GeneralRecords.js:205]:
```suggestion
        <Section title="Employment/skills summary" level={2} preserveWhiteSpace>
```

File: tgui/packages/tgui/interfaces/SecurityRecords.js (hunk 1 above)
Comment (ShadowLarkens) [inline tgui/packages/tgui/interfaces/SecurityRecords.js:182]:
```suggestion
              <Box height="20px" inline preserveWhiteSpace>
```

**Review by ShadowLarkens — CHANGES_REQUESTED** [review]

**Comment by Atermonera** [comment]
You'll have to update your branch with the updated CI OS version to get it to pass

**Pull request merged**